openforcefield/topology/topology.py

```diff
@@ -1851,6 +1851,38 @@ def to_openmm(self, ensure_unique_atom_names=True):
             omm_topology.setPeriodicBoxVectors(self.box_vectors)
         return omm_topology
 
+    def to_file(self, filename, positions, file_format="PDB", keepIds=False):
+        """
+        To save a PDB file with coordinates as well as topology from openforcefield topology object
+        Reference: https://github.com/openforcefield/openforcefield/issues/502
+        Note: 1. This doesn't handle virtual sites (they're ignored)
+              2. Atom numbering may not remain same, for example if the atoms in water are numbered as 1001, 1002, 1003,
+                 they would change to 1, 2, 3.
+                 This doesn't affect the topology or coordinates or atom-ordering in anyway
+              3. Same issue with the amino acid names in the pdb file, they are not returned
+
+        Parameters
+        ----------
+        filename : name of the pdb file to write to
+        positions : Can be an openmm 'quantity' object which has atomic positions as a list of Vec3s along with associated units, otherwise a 3D array of UNITLESS numbers are considered as "Angstroms" by default
+        file_format : write topology in pdb file format
```

Comment:
```suggestion
        filename : str
            name of the pdb file to write to
        positions : n_atoms x 3 numpy array or simtk.unit.Quantity-wrapped n_atoms x 3 iterable
            Can be an openmm 'quantity' object which has atomic positions as a list of Vec3s along with associated units, otherwise a 3D array of UNITLESS numbers are considered as "Angstroms" by default
        file_format : str
            Output file format. Case insensitive. Currently only supported value is "pdb".
```